tests/integration/testdata/accounts.go

```diff
@@ -24,7 +24,7 @@ const (
 	PERIODIC_VESTING_ACCOUNT_ADDR   = "cheqd1uyngr0l3xtyj07js9sdew9mk50tqeq8lghhcfr"
 
 	//? Module accounts can be safely hardcoded, as they are generated per module name
-	//? and the hex bytes are deterministicly generated from that module name.
+	//? and the hex bytes are deterministically generated from the module name.
```

Comment:
Is it deterministic for all networks, i.e., does it depend on chain ID? I think yes from the definition I see but want to double-check.